src/Components/ConfirmDialog/ConfirmDialog.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import "./ConfirmDialog.scss";
+
+import Button from "../Button/Button";
```

Comment:
add `.jsx` extension and also in other files for import sort plugin

Reply:
check